### DISCUSSION TITLE
Homepage daily widget/coding?

### DISCUSSION THREAD

**Discussion by LynnwoodAlanoClub** [code4recovery/12-step-meeting-list#1059] [General]
Hello, I was wondering if there is a way to add the list of meetings for the current day on the homepage as a widget? I just need a section of the homepage to show the list of the current day's meetings, I don't want the Meetings List to be set as the homepage. Or is there some coding I can add so that it'll show it this way?

I also wanted to ask how to keep custom Types permanently added? Somehow the ones I add always disappear after a short while. Types such as "Meeting Room A" or "Open" (without the subtext).

Thank you in advance!

**Comment by kiyote33** [comment]
Hi @LynnwoodAlanoClub,
Yes, there's a legacy widget for Upcoming Meetings which I've used in the past for an Alano Club. It can be tweaked to do what you're asking. Used with a child theme (functions.php) you could add some code there like 
$tsml_columns = array(
	'time' => 'Time',
	'distance' => 'Distance',
	'name' => 'Name',
 ); to restrict the columns that display to just the time and meeting name.

Your custom types should also be placed in your child theme so that they don't disappear whenever we make an update to the 12 Step Meeting LIst. I've successfully used a plugin a few times to create the child theme for my projects. That's one way to quickly handle that chore. Just look in the wordpress plugin directory for a child theme generator and you will probably find more than one.

Hope this helps...
